gravitee-policy-ratelimit/src/main/java/io/gravitee/policy/ratelimit/RateLimitKeyFactory.java

```diff
@@ -48,6 +48,14 @@ public static String createRateLimitKey(ExecutionContext executionContext, RateL
 
         StringBuilder key = new StringBuilder();
 
+        if (configuration.isUseKeyOnly() && configuration.getKey() != null && !configuration.getKey().isEmpty()) {
+            return key
+                .append(executionContext.getTemplateEngine().getValue(configuration.getKey(), String.class))
+                .append(KEY_SEPARATOR)
+                .append(RATE_LIMIT_TYPE)
+                .toString();
+        }
+
```

Comment:
probably overkill but it would have been cool to have a common module to handle key generation logic

Reply:
Yes I thought the same 🙈 but I didn't took the time to do it as I already spent some unscheduled time to re-enable tests 🙈